src/modules/auth/screens/login-screen.tsx

```diff
@@ -10,10 +10,10 @@ import {
   showNotification,
 } from '../../../ui/core/notifications/card-notification';
 import { signIn } from '../../../core';
-import { common } from '../../../translations/en.json'
+import { common } from '../../../translations/en.json';
 import { LogBox } from 'react-native';
 LogBox.ignoreLogs(['Warning: ...']); // Ignore log notification by message
-LogBox.ignoreAllLogs(true)
+LogBox.ignoreAllLogs(true);
```

Comment:
I generally turn this on for demos to avoid showing the clients the messages 😂  But I think for develop you should see the messages to see what is going wrong with warnings and errors 🙃

Reply:
Yes, I shouldn't ignore logs!

Reply:
done